ci/gpu/build.sh

```diff
@@ -84,14 +84,9 @@ logger "GoogleTest for libcuml..."
 cd $WORKSPACE/cpp/build
 GTEST_OUTPUT="xml:${WORKSPACE}/test-results/libcuml_cpp/" ./test/ml
 
-# Disabled while CI/the test become compatible
-# logger "GoogleTest for libcuml mg..."
-# cd $WORKSPACE/cpp/build
-# GTEST_OUTPUT="xml:${WORKSPACE}/test-results/libcuml_cpp_mg/" ./test/ml_mg
-
 logger "Python pytest for cuml..."
 cd $WORKSPACE/python
-pytest --cache-clear --junitxml=${WORKSPACE}/junit-cuml.xml -v
+pytest --cache-clear --junitxml=${WORKSPACE}/junit-cuml.xml -v -m "not mg"
```

Comment:
```suggestion
pytest --cache-clear --junitxml=${WORKSPACE}/junit-cuml.xml -v -m "sg"
```

More of a question, why `not mg` instead of `sg`?

Reply:
This is just how the pytests custom markers work. I've been marking the MG tests (since there are fewer of them) with `mg` so that we can run them in CI. In order to not run them during the regular CI, you use -m "not <margers_you_dont_want_to_run>". We could do it the opposite way, but that would require adding marks many more files.